src/EditorFeatures/CSharpTest2/Recommendations/NotKeywordRecommenderTests.cs

```diff
@@ -285,5 +293,25 @@ public async Task TestAfterRelationalOperator()
             await VerifyAbsenceAsync(AddInsideMethod(InitializeObjectE +
 @"if (e is >= 0 $$"));
         }
+
+        [WorkItem(61184, "https://github.com/dotnet/roslyn/issues/61184")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterIdentifierAnd()
+        {
+            await VerifyKeywordAsync(InitializeObjectE +
+@"if (e is Test.TestValue and $$)
+
+enum Test { TestValue }");
+        }
```

Comment:
💡 A `Theory` here eliminates the need for multiple tests that have the same basic form.

```suggestion
        [WorkItem(61184, "https://github.com/dotnet/roslyn/issues/61184")]
        [Theory, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
        [InlineData("and")]
        [InlineData("or")]
        [InlineData("not")]
        public async Task TestAfterIdentifierAnd(string precedingKeyword)
        {
            await VerifyKeywordAsync(InitializeObjectE +
$@"if (e is Test.TestValue {precedingKeyword} $$)

enum Test {{ TestValue }}");
        }
```